azure-pipelines.yml

```diff
@@ -33,9 +33,15 @@ jobs:
         inputs:
           versionSpec: "3.8"
 
-      - bash: "sudo apt update && sudo apt-get install --yes default-jre default-jdk npm"
+      - bash: "sudo apt update && sudo apt-get install --yes default-jre default-jdk"
         displayName: Install apt dependencies
 
+      - bash: |
+          wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+          curl -sL https://deb.nodesource.com/setup_13.x | sudo -E bash -
+          sudo apt-get install -y nodejs
+        displayName: Install Node
```

Comment:
This won't work, it'll be regarded as a part of the multiline string. You might have to inline the bash value to get it to work